meetings/2015-03-09/minutes.md

```diff
@@ -0,0 +1,144 @@
+**TF:** To clarify, there is a draft that will be circulated after this meeting. The Node.js team will schedule a call this week to discuss and draft will be sent out. Working on creating Technical Governance for foundation and creating a solid path forward for Node.  
+**BB:** May I suggest that the proposal goes on the internet somewhere.  
+**GR:** Who is on the Node.js committee?  
+**BB:** Participants from Joyent, Microsoft, and IBM. [Additional names provided, but not recorded.]  
+**GR:** Would love to see progress. I think io.js proposal is a good starting point and would like to understand what is on the other side. Lets make sure that there’s a good timeline in mind.  
+**TF:** Draft will be sent around after this meeting.  
+**DS:** Any thoughts on current io.js proposal?  
+**DC:** Some items WRT trademarks are something I haven’t seen before. For example, TC doesn’t typically own or manage trademark. Board manages that and issues resolved through discussion.  
+**IS:** Can we get that feedback on the proposal such that we can iterate on it?  
+**DS:** Yeah, that’s an easy solve.  
+**SH:** I feel like the main issue is the TC decision. I would rather segregate this and let the Linux Foundation do their thing and have the TC conversation separately.  
+**JZ:** We’re down to a fairly narrow set of issues, so let’s stay focused on that.  
+**IR:** The response [Node.js proposal] should have been released prior to the meeting.  
+**TF:** The concern was that this meeting would get derailed on details of the proposals.  
+**IS:** So there are things that should belong on the business side. We have not aligned on issues, but said issues have not been identified.  
+**SH:** We need to have conversations between TF and BB to make progress.  
+**BB:** “Progress” is an open concept. MD tried to write down what TF wants. It was an attempt to identify what TF wants, but it sounds like the true proposal will be circulated.  
+**TF:** New document provides additional context.  
+**CW:** We need to identify ONE owner to drive this for the next meeting.  
+**JZ:** There is progress. Let’s not be so hard on each other. LF will continue to drive. We will continue to broker a convergence. Will refine external communication, but progress is being made. Let’s move forward with current plan.  
+**BB:** In order to get community support we have to do it in the open, meaning we open up the process, not the outcome. This is needed to build trust.  
+**JZ:** I agree.  
+**BB:** We need to discussion in the open and will cross-post in io.js.  
+**IS:** Any reconciliation announcement needs to have a link back to the discussion.  
+
+
+## New Action Items
+**TF:** Send Node.js Technical governance proposal out to interested parties.  
+
```

Comment:
I would like to see the proposal. But why not put it in a github issue like iojs did, so everybody can see it?